extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleDistributionManager.java

```diff
@@ -103,7 +103,7 @@ public final class GradleDistributionManager {
         GradleVersion.version("8.5"), // JDK-21
     };
 
-    private static final GradleVersion LAST_KNOWN_GRADLE = GradleVersion.version("8.6"); //NOI18N
+    private static final GradleVersion LAST_KNOWN_GRADLE = GradleVersion.version("8.7"); //NOI18N
```

Comment:
looks like groovy is holding up gradle again:
https://docs.gradle.org/8.7/release-notes.html#java-22

Reply:
Damn! Why did no one tell groovy that this should not work:

```
matthias@enterprise:~/Downloads$ ~/bin/groovy-4/bin/groovy -v
Groovy Version: 4.0.20 JVM: 22 Vendor: Oracle Corporation OS: Linux
matthias@enterprise:~/Downloads$ ~/bin/groovy-4/bin/groovy -e 'print "Hello World\n"'
Hello World
matthias@enterprise:~/Downloads$
```

http://groovy-lang.org/changelogs/changelog-4.0.16.html references support for JDK 22. That release was created about 4 months ago.

I think this might be a bit more involved, than "groovy does not support JDK 22", maybe it is "we did not manage to integrate the groovy version that support JDK 22".

Reply:
i think the release notes are a bit oversimplified which caused issues in past since they were not clear what is supposed to work and what doesn't work on current jdks.

https://github.com/gradle/gradle/issues/26162 this issue tracks JDK 22 progress. Looking at it its appears that Kotlin is holding them back and not Groovy.